invenio_app_ils/cli.py

```diff
@@ -79,6 +84,11 @@ def get_documents_items(internal_locations, n_docs, n_items):
             "title": "{}".format(lorem.sentence()),
             "authors": ["{}".format(lorem.sentence())],
             "abstracts": ["{}".format(lorem.text())],
+            "document_types": [random.choice(DOCUMENT_TYPES)],
```

Comment:
Are you sure this is plural in the document schema or it's not defined yet? I remember in INSPIRE having it as `document_type`...

Reply:
All the other attributes which are a list have plural so I sticked to this convention, so maybe we should align on one strategy for naming.

Reply:
IMHO, there should be no exceptions, all should follow the same convention.